extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcSession.java

```diff
@@ -16,12 +17,21 @@ public interface OidcSession {
     String getTenantId();
 
     /**
-     * Return an {@linkplain:Instant} indicating how long will it take for the current session to expire.
+     * Return an {@linkplain Instant} representing the current session's expiration time
+     * which is a number of seconds from the epoch of 1970-01-01T0:0:0Z.
      *
-     * @return
+     * @return Instant
      */
     Instant expiresIn();
 
+    /**
+     * Return a {@linkplain Duration} indicating how long will it take for the current session to expire
+     * starting from the current time
```

Comment:
Maybe instead of 'current time' we should say invocation time?

Reply:
@geoand Yeah, sure, I was contemplating if we should imply that this check is done at the request (invocation) time, I can update

Reply:
@geoand I've changed it to `starting from this method's invocation time` as I thought it might be open to a few interpretations otherwise what invocation time is referred to here... Are you OK with it ?

Reply:
Looks like so as you've just approved :-)